src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpBuild.cs

```diff
@@ -35,5 +37,25 @@ static void Main(string[] args)
             // TODO: Validate build works as expected
         }
 
+        [Fact]
+        public void BuildWithCommandLine()
+        {
+            VisualStudio.Instance.SolutionExplorer.SaveAll();
+
+            var pathToDevenv = Path.Combine(VisualStudio.Instance.InstallationPath, @"Common7\IDE\devenv.exe");
+            var pathToSolution = VisualStudio.Instance.SolutionExplorer.SolutionFileFullPath;
+            var logFileName = pathToSolution + ".log";
+
+            File.Delete(logFileName);
```

Comment:
@jaredpar also if it doesn't exist, or does exist. Logically, all are valid antecedents. :pedantic pendantry: 😛